### ISSUE TITLE
Web warnings with v2.1.0

### ISSUE THREAD

**Issue by Hirbod** [software-mansion/react-native-gesture-handler#1769]
## Description

After upgrading to Expo SDK 44 (from 43), I get a lot of warnings in my console on web. This wasn't the case before.
It doesn't matter if I import RNGH, the logs are spammed anyway. Also tried to wrap into a GestureHandlerRootView but the warnings still appear in my webbrowser.

### Screenshots

<img width="931" alt="Bildschirmfoto 2021-12-17 um 13 19 53" src="https://user-images.githubusercontent.com/504909/146544069-efac9a72-491a-4113-a9e3-e44e9111feed.png">

## Steps To Reproduce

1. Install RNGH.
2. Run on Web
3. Watch console

### Expected behavior
No warnings in console

### Actual behavior
Warnings in console.

```
  Expo CLI 5.0.1 environment info:
    System:
      OS: macOS 11.6.1
      Shell: 5.8 - /bin/zsh
    Binaries:
      Node: 14.18.2 - /usr/local/opt/node@14/bin/node
      Yarn: 1.22.17 - /usr/local/bin/yarn
      npm: 6.14.15 - /usr/local/opt/node@14/bin/npm
    Managers:
      CocoaPods: 1.11.2 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.2, iOS 15.2, macOS 12.1, tvOS 15.2, watchOS 8.3
    IDEs:
      Android Studio: 4.1 AI-201.8743.12.41.6953283
      Xcode: 13.2/13C90 - /usr/bin/xcodebuild
    npmPackages:
      @expo/metro-config: ^0.3.4 => 0.3.4 
      @expo/webpack-config: ~0.16.2 => 0.16.14 
      expo: ^44.0.0 => 44.0.0 
      react: 17.0.1 => 17.0.1 
      react-dom: 17.0.1 => 17.0.1 
      react-native: 0.64.3 => 0.64.3 
      react-native-web: 0.17.1 => 0.17.1 
    npmGlobalPackages:
      eas-cli: 0.33.0
      expo-cli: 5.0.1
    Expo Workflow: managed
```
- RNGH: 2.1.0

**Comment by Maker-Mark** [comment]
@bfischer1121 Do you know when this will be available on npm?

**Comment by bfischer1121** [comment]
hmm no idea -- upgrading wasn't a big priority for me so I just stuck with 1.x. Worth noting that this breaks development in vite projects (actual errors when precompiling deps), so it'd be nice to get a new release to fix it cc: @j-piasecki

**Comment by j-piasecki** [comment]
Hi! The just released GH 2.2.0 includes https://github.com/software-mansion/react-native-gesture-handler/pull/1791 which fixes this issue.
